test/controllers/supervision/ProfessionalBodyMemberControllerSpec.scala

```diff
@@ -96,7 +153,22 @@ class ProfessionalBodyMemberControllerSpec extends AmlsSpec with MockitoSugar {
           }
         }
       }
+
       "redirect to PenalisedByProfessionalController" when {
+
+        "the answer is no to the question and no previous professional bodies has been registered in the cache" in new Fixture {
```

Comment:
Suggestion;

'the answer to the question is no and no previous professional bodies exist in the cache'